Admin/Custom.Xperience.Stripe.Admin/Constants/XperienceStripeConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Custom.Xperience.Stripe
+{
```

Comment:
(Commenting here because I can't comment on the related file) The XperienceStripeService should use injection for IOrderItemInfoProvider, ILocalizationService (instead of ResHelper), ICurrencyInfoProvider